src/components/Alert/example.js

```diff
@@ -65,31 +60,6 @@ export default class AlertExample extends React.Component {
             Error: Wow you really screwed this up...
           </Alert>
         </Example>
-        <Example>
```

Comment:
Why are the multiline and multiline+title examples being removed? Seems separate to this PR?

Reply:
I couldn't find anywhere explicitly using the multiline feature so maybe fine to remove. I also couldn't find anywhere we use the title but I think having an example showing the title is worthwhile. 

Could maybe remove just the multiline and change the title one to be a `warning` or `error` to align with the warning icon?

Reply:
> and didn't want people to take that as an example of something to do

Are we saying multiline alerts should not be used?

+1 on title/icon.